.ci/scripts/clang-format.sh

```diff
@@ -1,7 +1,7 @@
 #! /bin/bash
 
 #===============================================================================
-# Copyright 2020 Intel Corporation
+# Copyright 2021 Intel Corporation
```

Comment:
It should be "2020-2021"

Reply:
For **cpp/daal/src/algorithms/dtrees/forest/regression/oneapi/df_regression_train_hist_oneapi_impl.i**  also should be "2020-2021" ?

Reply:
For every file, "2020" should become "2020-2021", "...-2020" should become "...-2021"